CopilotKit/packages/react-core/src/components/copilot-provider/copilotkit.tsx

```
@@ -380,6 +385,181 @@ export function CopilotKitInternal(cpkProps: CopilotKitProps) {
+
+        case "network":
+          console.error(`[CopilotKit Network Error] ${categorizedError.message}`, {
+            type: categorizedError.type,
+            endpoint: categorizedError.endpoint,
+            statusCode: categorizedError.statusCode,
+            threadId: categorizedError.threadId,
+          });
+          break;
+
+        case "component":
+          const isExtractError = categorizedError.componentName === "extract";
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)